README.md

```diff
@@ -49,11 +49,27 @@ There are 2 pieces of code that need to be included in your webpage:
     }
   };
 
+  SURF_N_PERF.setFirstPaint = function() {
+    SURF_N_PERF.marks.firstPaintFrame = (new Date()).getTime();
+
+    if(window.performance) {
+    if(window.performance.now) {
```

Comment:
This can just be one line: `if(window.performance && window.performance.now) {`
In JavaScript, if the first statement before the `&&` is falsy, it won't try to evaluate the second